server/src/test/java/io/crate/expression/scalar/arithmetic/SquareRootFunctionTest.java

```diff
@@ -39,12 +41,16 @@ public void testSqrt() {
         assertEvaluateNull("sqrt(null)");
         assertEvaluate("sqrt(cast(25 as integer))", 5.0);
         assertEvaluate("sqrt(cast(25.0 as float))", 5.0);
+        assertEvaluate("sqrt(cast(123.4 as numeric(6, 1)))",
+            new BigDecimal("11.10855526159905278255972911272118"));
```

Comment:
Shouldn't it have less digits?

Reply:
postgres doesn't seem to respect the input arg, and truncate accordingly the result:
```
matriv=> select sqrt(cast(123.4 as numeric(4, 1)));
        sqrt        
--------------------
 11.108555261599053
(1 row)
```
and imho we should not preserve the type for `sqrt` and `exp`. It's kind of the same behavior that those 2 functions when they take an `int` arg, the return `double`, the highest available precision type we have. So for numeric of any precision/scale we should probably return just `DataTypes.NUMERIC` with null precision and scale.

Reply:
> On the contrary, I believe, abs/ceil/floor/round should preserve it.

Could we also consider not preserving precision/scale for Postgres compatibility? Or for consistency across numeric operations.

Reply:
We could also do that, yes.

Reply:
As discussed, we chose, for the time being, to return generic `NUMERIC` without precision/scale, for all those functions.

Reply:
Added comment to relevant issue, to revisit this behavior consistently: https://github.com/crate/crate/issues/16478#issuecomment-2426200103